core/float/round_spec.rb

```diff
@@ -10,11 +10,9 @@
     0.0.round.should == 0
   end
 
-  platform_is_not "x64-mingw32" do
-    it "returns the nearest Integer for Float near the limit" do
-      0.49999999999999994.round.should == 0
-      -0.49999999999999994.round.should == 0
-    end
+  it "returns the nearest Integer for Float near the limit" do
```

Comment:
This is a bug in mingw runtime, and we should report it to the upstream, I think.
Once it gets fixed there, will we need a guard for mingw version?